### PR TITLE
Feature: Open file to change when in inline diff

### PR DESCRIPTION
**Pull request by stoivo**
I created the feature to open file you that is open in inline diff.
Currently it is only for while you are in a hunk, but it should be implemented to do this for outside as well. 
If you have a good idea how to make this easily, I would like to hear it ✨ 

closes divmain/GitSavvy#236

### CODE REVIEW

File: core/commands/inline_diff.py [hunk 1]
```diff
@@ -579,6 +579,48 @@ def get_diff_from_line(self, line_no, reset):
 
         return "\n".join([stand_alone_header] + hunk_ref.hunk.raw_lines[1:])
 
+class GsInlineDiffOpenFile(TextCommand):
+
+    """ Should open file at current hunk. """
+
+    def run(self, edit):
+        selections = self.view.sel()
+        region = self.view.line(0) if len(selections) == 0 else selections[0]
```
Comment (divmain) [inline core/commands/inline_diff.py]: Nitpick: please change this to:
```python
region = selections[0] if selections else sublime.Region(0, 0)
```

File: core/commands/inline_diff.py [hunk 2]
```diff
@@ -579,6 +579,48 @@ def get_diff_from_line(self, line_no, reset):
...
+
+        # Git lines are 1-indexed; Sublime rows are 0-indexed.
+        current_line_number = self.view.rowcol(region.begin())[0] + 1
+        current_column_number = self.view.rowcol(region.begin())[1] + 1
```
Comment (divmain) [inline core/commands/inline_diff.py]:
```python
line_no, column_no = self.view.rowcol(region.begin())
# Git lines are 1-indexed; Sublime rows are 0-indexed.
line_no, column_no = line_no + 1, column_no + 1
```

**Comment by divmain** [comment]
I pulled in your commit and merged after making changes to support any cursor position.  Thanks @stoivo!

**Comment by stoivo** [comment]
Do you know when you will make a new release?

**Comment by divmain** [comment]
@stoivo, usually I cluster features together.  Mostly, I do this to avoid irritating users with a new release and pop-up message every other day.  But I do release if somebody asks for it.

I hope you don't mind that I made significant changes.  I started commenting on the PR and realized it would be much quicker to just write the code.  There was a rebase, but the first commit is still attributed to you.

**Comment by stoivo** [comment]
I totally agree that is stupid to place a release every other day. I just ask in case you had any schedule for releases.  

I read through your commit thoroughly so I can learn from it since you are really good at writing python. One day, I will get closer to write good python :+1: I don't mind you correcting me ;)

**Comment by divmain** [comment]
Yeah, I don't have a formal release schedule, mostly because development velocity is inconsistent. Also, I'll just say your code is definitely good, and only limited when it comes to some of the language-specific syntax and idioms.

**Comment by stoivo** [comment]
Definitely, I only learn python by reading some small plugins, but mostly this repo. And Thank you!!

**Pull request closed without merging**